src/rdb/tests/rpc.h

```diff
@@ -74,12 +113,75 @@ CRT_RPC_DECLARE(rdbt_init, DAOS_ISEQ_RDBT_INIT_OP, DAOS_OSEQ_RDBT_INIT_OP)
 
 CRT_RPC_DECLARE(rdbt_fini, DAOS_ISEQ_RDBT_FINI_OP, DAOS_OSEQ_RDBT_FINI_OP)
 
+#define DAOS_ISEQ_RDBT_PING_OP	/* input fields (none) */
+
+#define DAOS_OSEQ_RDBT_PING_OP	/* output fields */		 \
+	((struct rsvc_hint)	(tpo_hint)		CRT_VAR) \
+	((int32_t)		(tpo_rc)		CRT_VAR)
+
+CRT_RPC_DECLARE(rdbt_ping, DAOS_ISEQ_RDBT_PING_OP, DAOS_OSEQ_RDBT_PING_OP)
+
+#define DAOS_ISEQ_RDBT_CREATE_OP	/* input fields (none) */
+
+#define DAOS_OSEQ_RDBT_CREATE_OP	/* output fields */	 \
+	((struct rsvc_hint)	(tco_hint)		CRT_VAR) \
+	((int32_t)		(tco_rc)		CRT_VAR)
+
+CRT_RPC_DECLARE(rdbt_create, DAOS_ISEQ_RDBT_CREATE_OP,
+		DAOS_OSEQ_RDBT_CREATE_OP)
+
+#define DAOS_ISEQ_RDBT_DESTROY_OP	/* input fields (none) */
+
+#define DAOS_OSEQ_RDBT_DESTROY_OP	/* output fields */	 \
+	((struct rsvc_hint)	(tdo_hint)		CRT_VAR) \
+	((int32_t)		(tdo_rc)		CRT_VAR)
+
+CRT_RPC_DECLARE(rdbt_destroy, DAOS_ISEQ_RDBT_DESTROY_OP,
+		DAOS_OSEQ_RDBT_DESTROY_OP)
+
 #define DAOS_ISEQ_RDBT_TEST_OP	/* input fields */		 \
-	((int32_t)		(tti_update)		CRT_VAR)
+	((int32_t)		(tti_update)		CRT_VAR) \
+	((int32_t)		(tti_memb_op)		CRT_VAR) \
+	((uint64_t)		(tti_key)		CRT_VAR) \
+	((uint64_t)		(tti_val)		CRT_VAR)
 
 #define DAOS_OSEQ_RDBT_TEST_OP	/* output fields */		 \
+	((struct rsvc_hint)	(tto_hint)		CRT_VAR) \
+	((uint64_t)		(tto_val)		CRT_VAR) \
 	((int32_t)		(tto_rc)		CRT_VAR)
 
 CRT_RPC_DECLARE(rdbt_test, DAOS_ISEQ_RDBT_TEST_OP, DAOS_OSEQ_RDBT_TEST_OP)
 
+#define DAOS_ISEQ_RDBT_MEMBERSHIP /* input fields */		 \
+	((d_rank_list_t)	(rtmi_ranks)		CRT_PTR)
+
+#define DAOS_OSEQ_RDBT_MEMBERSHIP /* output fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
False positive for CaRT RPC definition related macros